backend/handlers/handlers.go

```diff
@@ -874,9 +877,10 @@ func (handlers *Handlers) getAccountSummary(_ *http.Request) (interface{}, error
 			string(account.Coin().Code()),
 			fiat)
 		earliestTxTime := txs.EarliestTime()
-		// TODO: show chart starting at `earliestPriceavailable`, so the last three months can be
-		// shown more quickly instead of waiting for all historical rates first.
-		// For now, we show the chartDataMissing error.
+		if earliestTxTime.IsZero() {
+			// Ignore the chart for this account, there is no timed transaction.
+			continue
+		}
 		if earliestPriceAvailable.IsZero() ||
 			(!earliestTxTime.IsZero() && earliestTxTime.Before(earliestPriceAvailable)) {
```

Comment:
Since you already check for `if earliestTxTime.IsZero()` above now, this can be simplified to:

```go
if earliestPriceAvailable.IsZero() || earliestTxTime.Before(earliestPriceAvailable)
```

easier to read.

Reply:
Excellent! Done.